upgrade/UPGRADE-11.0.0-dev.md

```diff
@@ -0,0 +1,6 @@
+# [Upgrade from v10.0.0 to v11.0.0-dev](https://github.com/shopsys/shopsys/compare/10.0.0...master)
```

Comment:
shouldn't this file be named with a `v` prefix before the version?

```diff
-  UPGRADE-11.0.0-dev.md
+  UPGRADE-v11.0.0-dev.md
```

Reply:
yeah, it seems like there are more places where `getOriginalString` should be used instead of `getVersionString`, however, I will not delay the release process by fixing it now, I will simply change the filename manually now and put this problem into a user story as well.

6cf41cede